src/lib/openapi/util/create-response-schema.ts

```diff
@@ -36,6 +36,14 @@ export const createResponseSchema = (
     });
 };
 
+export const createCsvResponseSchema = (
+    schemaName: string,
+): OpenAPIV3.ResponseObject => {
+    return createResponseSchemas(schemaName, {
+        'text/csv': schemaNamed(schemaName),
```

Comment:
Cool! Does this convert it to CSV (for examples etc)? Does the example response show as CSV in swagger UI?

Reply:
Unfortunately swagger does not show examples for CSV

Reply:
Hmm ... Yeah, it doesn't look like it's well supported. Can we still show examples, though? according to this stack overflow thread, you should [mark CSV payloads as strings](https://stackoverflow.com/questions/59722686/how-to-properly-document-text-csv-responses-on-swagger-yaml-docs) if I read correctly. And they can probably have examples. Right?

Reply:
This is annoying, the schema still shows the json example, but we definitely have an object here, so I'm inclined to keep it.

Reply:
yeah, which means we should just drop the `schemaNamed` here then. and just use string

Reply:
I fixed it by making the type of the response string, and then allowing the makeCsvResponseSchema take an example string. Our example is now a csv serialized object of the type we're expecting.